webapp/config/packages/doctrine.yaml

```diff
@@ -24,6 +24,7 @@ doctrine:
         #server_version: '15'
     orm:
         auto_generate_proxy_classes: true
+        enable_lazy_ghost_objects: false
```

Comment:
What does this do? According to google this would stop us from having the objects in memory but I assume that is what we want?

Reply:
It’s a new feature which has a bug and breaks stuff. The lazy ghosts is a fancy thing that will auto load relations only when you need it (even more than normal), but it has an open bug we run into